lib/node_modules/@stdlib/stats/base/dists/gumbel/skewness/src/main.c

```diff
@@ -0,0 +1,67 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// MODULES //
+
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/sqrt.h"
+#include "stdlib/math/base/special/riemann_zeta.h"
+#include "stdlib/constants/float64/pi.h"
+
+
+// MAIN //
+
+/**
+* Returns the skewness for a Gumbel distribution with location `mu` and scale `beta`.
+*
+* @param {number} mu - location parameter
+* @param {PositiveNumber} beta - scale parameter
+* @returns {number} skewness
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( 0.0, 1.0 );
+* // returns ~1.14
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( 5.0, 2.0 );
+* // returns ~1.14
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( NaN, 1.0 );
+* // returns NaN
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( 0.0, NaN );
+* // returns NaN
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( 0.0, 0.0 );
+* // returns NaN
```

Comment:
By convention, we only keep one example in the C comments:

```suggestion
* @example
* double y = stdlib_base_dists_gumbel_skewness( 0.0, 1.0 );
* // returns ~1.14
```